changelog/13022.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+implements Login method in Go client libraries for GCP and Azure auth methods
```

Comment:
Just a note for next time - we'd want this file to have a submodule as a prefix to this line - in this case, it would be something like: `api: Implement Login method in...`

I'll tag you as a reviewer on an ENT PR so the changelog will format properly.

Reply:
Ah, sorry about that!